src/components/reference/button-container.tsx

```diff
@@ -1,12 +1,12 @@
-import * as React from 'react';
```

Comment:
I think you can probably kill this file now.

Reply:
might keep around for a bit longer just as reference, but eventually I will.